baselines/Google Drive and Docs Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -98,10 +98,11 @@ Agencies SHALL disable making files and published web content visible to anyone
   - [T1537: Transfer Data to Cloud Account](https://attack.mitre.org/techniques/T1537/)
 
 #### GWS.DRIVEDOCS.1.6v0.2
-Agencies SHALL enable access checking for file sharing outside of Docs or Drive.
+Agencies SHOULD set access checking to recipients only.
 
 - _Rationale:_ The Access Checker feature can be configured to allows users to grant access to the public if a recipient is missing access, creating the potential for data leakage. This control mitigates this by only allowing access to be granted to recipients or the suggested target audience.
-- _Last modified:_ July 10, 2023
+- _Last modified:_ June 7, 2024
+- _Note:_ Agencies SHALL NOT set access checking to Recipients only, suggested target audience, or public (no Google account required).
```

Comment:
We shouldn't have a normative requirement, especially a SHALL, in a note. Notes are for informational bits, like clarifications or exceptions. Furthermore, ScubaGoggles isn't capable of checking polices that are simultaneously SHALL and SHOULD. On M365, we've gone through the work of decoupling all such policies (just a few left, for example see: https://github.com/cisagov/ScubaGear/issues/524). We shouldn't start adding them here.

Furthermore, I don't think the "suggested target audience" option is necessarily a bad thing. At first glance it seems like a very hand-wavy thing, leaving you wondering how Google determines who is in your target audiences; if that were the case I would agree that we should recommend against it. But it turns out that target audiences are actually a very precise thing that you have to configure. See [About target audiences](https://support.google.com/a/answer/9934697?hl=en.) The bottom line: you decide who goes into a target audience.
And per [Add or remove target audience members](https://support.google.com/a/answer/9934621):
- Non-Google users can't be added to target audience
- External users can only be added to a target audience if Drive allows sharing outside your organization (i.e., GWS.DRIVEDOCS.1.1v0.2).
Those already built-in requirements for target audiences minimizes the risk they present.

Therefore, I'd recommend taking this note and making it the main policy for GWS.DRIVEDOCS.1.6v0.2. Though if we did care enough about target audiences to include a soft recommendation against them here, this SHALL NOT statement should be split out to into its own policy.

Finally, currently the wording for the SHALL NOT statement can be read as listing 3 different things that access checking shouldn't be set to, i.e., "don't set it to recipients only, also don't set it to suggested target audience, also don't set it to public." Wrapping "Recipients only, suggested target audience, or public (no Google account required)" in quotes would make it more clear that we're referring to a single option. But that introduces the problem of using the precise "buttonology" in the policy statement, something we try to avoid to make the policies less brittle, especially with a string that long. The real important bit is that they shouldn't set it to public. One potential way of wording it that fixes these issues (and as an added bonus clarifies what this setting actually does):
>Access Checking SHALL NOT suggest sharing to the public (no access required).

Reply:
We will take a look at it. However, "Recipients only, suggested target audience, or public (no Google account required)" is not 3 things it is 1 option in Google Workspace Admin Console. If you look at the setting all of these are listed in 1 selection option. @adhilto Please make sure to look more closely before making a comment about something.

Reply:
> However, "Recipients only, suggested target audience, or public (no Google account required)" is not 3 things it is 1 option in Google Workspace Admin Console

@jkaufman-mitre That's my exact point. The way it's currently worded in the baseline makes it sound like it could be 3 things. My suggested rewording removes that ambiguity.